code/src/component/ToDos.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useState } from "react";
+import { useSelector } from "react-redux";
+import ToDo from "./ToDo";
+
+const ToDos = () => {
+  const todos = useSelector((state) => state.todo.todo);
+  const [todoList, setTodoList] = useState([]);
+
+  useEffect(() => {
+    setTodoList(todos);
+  }, [todos]);
+
+  const completed = todos.filter((todo) => todo.isCompleted).length;
+
+  const showCompleted = () => {
+    const completedList = todos.filter((todo) => todo.isCompleted);
+    setTodoList(completedList);
+  };
+
+  const showInCompleted = () => {
+    const incompletedList = todos.filter((todo) => !todo.isCompleted);
+    setTodoList(incompletedList);
+  };
+
+  return (
+    <>
+      <ul className="container-column" style={{ width: "100%" }}>
+        {todoList.map((todo) => (
+          <ToDo key={todo.id} todoItem={todo} />
+        ))}
+      </ul>
+      <article className="container-center-space-between sort">
+        <p>Sort by : &nbsp;</p>
+        <button className="btn-sort" onClick={() => setTodoList(todos)}>
+          All [{todos.length}]
+        </button>
+        <button className="btn-sort" onClick={showCompleted}>
+          Completed [{completed}]
+        </button>
+        <button className="btn-sort" onClick={showInCompleted}>
+          Ongoing [{todos.length - completed}]
```

Comment:
Impressed how clean and "simple" you manage to add number of both completed and uncompleted tasks.